srcpkgs/termite/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'termite'
+pkgname=termite
+version=14
+revision=1
+_utilcommit=62faf9e
+build_style=gnu-makefile
+hostmakedepends="ncurses pkg-config"
+makedepends="vte3-devel"
+depends="termite-terminfo"
+short_desc="A keyboard-centric VTE-based terminal"
+maintainer="Christian Buschau <christian.buschau@mailbox.org>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/thestinger/termite"
+distfiles="https://github.com/thestinger/termite/archive/v${version}.tar.gz>termite-${version}.tar.gz
+ https://github.com/thestinger/util/archive/${_utilcommit}.tar.gz>util-${_utilcommit}.tar.gz"
+checksum="0ef2ffaf2c7490b958ead45ff93dd5618636cfb5a5449ce4c87d068c748fe722
+ c62be725845c52751d9bd7a8defaff9ad1b29c473c0c279ad64f2cc59befaf2b"
+
+post_extract() {
+	rmdir ${wrksrc}/util
+	ln -s ../util-${_utilcommit}* ${wrksrc}/util
```

Comment:
You should make sure that there are no files created outside of `${wrksrc}` since the won't be cleaned up automatically. So moving the content of `util-${_utilcommit}*/` into `${wrksrc}` is probably the easiest way to do it.